vcsclient/bitbucketcloud.go

```diff
@@ -16,21 +15,29 @@ import (
 
 type BitbucketCloudClient struct {
 	vcsInfo VcsInfo
+	url     *url.URL
```

Comment:
Why a pointer?

Reply:
To check if the value is initialized in `buildBitbucketCloudClient`: 
```go
if client.url != nil {
	bitbucketClient.SetApiBaseURL(*client.url)
}
```
Do you see any problem with that?

Reply:
No,OK, I get it. It makes sense if there's a default value in the bitbucket client library.